### PR TITLE
Update UG to fit updated find-b/s, clear-b/s/all and list-b/s commands

### PR DESCRIPTION
**Pull request by cwnm**
Also, fixed formatting for user stories

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -92,6 +92,12 @@ Format: `exit`
 
 [back to start of section](#features)
 
+#### Clear. `clear-all`
+
+Function: Clears both buy er and seller lists
```
Comment (Jacky142857) [inline docs/UserGuide.md:97]: white space between

Comment (janald99) [inline docs/UserGuide.md]: typo here

**Review by janald99 — CHANGES_REQUESTED** [review]
just some tweaks before merging

File: docs/UserGuide.md (hunk 1 above)
Comment (cwnm) [reply docs/UserGuide.md]: thanks for the catch

Comment (cwnm) [reply docs/UserGuide.md]: edit: added warning to clear-all

**Review by janald99 — APPROVED** [review]
LGTM

**Pull request merged**